templates/multi-app/csharp/service/.gitignore

```diff
@@ -0,0 +1,5 @@
+flutter/
+.vs/
+*.user
+bin/
+obj/
```

Comment:
Is the .NET template in `multi-app/csharp/` currently supported?

I suppose that more work is required to support plugin registration since `GeneratedPluginRegistrant.cs` is not currently part of the C# project.

Reply:
You are right, C# is not yet supported. I will just remove this template.